backend-proxy-remote.conf.js

```diff
@@ -1,5 +1,10 @@
 let backendhostname = process.env.RUNBOX7_ANGULAR_BACKEND_HOST;
-if(!backendhostname) {
+if(backendhostname) {
+    // If we're using a custom development backend, turn off security checking to avoid certificate errors:
+    backendhostsecure = false;
```

Comment:
I think now we never have it set to `true`: it's either `false` from this is or undefined – which is probably also false.

Maybe it should be predeclared up top, after backendhostname? It can be false initially, then set to true if we get to the default production case.